jfrog-cli/artifactory/utils/golang/cachevalues_test.go

```diff
@@ -0,0 +1,133 @@
+package golang
+
+import (
+	"math/rand"
+	"testing"
+)
+
+func TestSuccessValues(t *testing.T) {
+	global1 := GetStaticCache()
+	var success int
+	global1.IncreaseSuccess()
+	success++
+	global2 := GetStaticCache()
+	if global2.GetSuccess() != success {
+		t.Error("Expected to get", success, ", got:", global2.GetSuccess())
+	}
+
+	global2.IncreaseSuccess()
+	success++
+	global2.IncreaseSuccess()
+	success++
+	global2.IncreaseSuccess()
+	success++
+
+	if global1.GetSuccess() != success {
+		t.Error("Expected to get", success, ", got:", global2.GetSuccess())
+	}
+
+	randValue := rand.Intn(100)
+
+	i := 0
+	for i < randValue {
+		global1.IncreaseSuccess()
+		success++
+		i++
+	}
+
+	if global2.GetSuccess() != success {
+		t.Error("Expected to get", success, ", got:", global2.GetSuccess())
+	}
+}
+
+func TestFailureValues(t *testing.T) {
+	global1 := GetStaticCache()
+	var failure int
+	global1.IncreaseFailures()
+	failure++
+	global2 := GetStaticCache()
+	if global2.GetFailures() != failure {
+		t.Error("Expected to get", failure, ", got:", global2.GetFailures())
+	}
+
+	global2.IncreaseFailures()
+	failure++
+	global2.IncreaseFailures()
+	failure++
+	global2.IncreaseFailures()
+	failure++
+
+	if global1.GetFailures() != failure {
+		t.Error("Expected to get", failure, ", got:", global2.GetFailures())
+	}
+
+	randValue := rand.Intn(100)
+
+	i := 0
+	for i < randValue {
+		global1.IncreaseFailures()
+		failure++
+		i++
+	}
+
+	if global2.GetFailures() != failure {
+		t.Error("Expected to get", failure, ", got:", global2.GetFailures())
+	}
+}
+
+func TestTotalValue(t *testing.T) {
+	global1 := GetStaticCache()
+	var total int
+	global1.IncreaseTotal(1)
+	total++
+	global2 := GetStaticCache()
+	if global2.GetTotal() != total {
+		t.Error("Expected to get", total, ", got:", global2.GetTotal())
+	}
+
+	global2.IncreaseTotal(1)
+	total++
+	global2.IncreaseTotal(1)
+	total++
+	global2.IncreaseTotal(1)
+	total++
+
+	if global1.GetTotal() != total {
+		t.Error("Expected to get", total, ", got:", global2.GetTotal())
+	}
+
+	randValue := rand.Intn(100)
+	global3 := GetStaticCache()
+	global3.IncreaseTotal(randValue)
+	total += randValue
+	if global2.GetTotal() != total {
+		t.Error("Expected to get", total, ", got:", global2.GetFailures())
+	}
+}
+
+func TestGlobalMap(t *testing.T) {
```

Comment:
This test function can be probably removed.

Reply:
Done